examples/rest-api-server.py

```diff
@@ -18,11 +18,11 @@
 from neo.Implementations.Notifications.LevelDB.NotificationDB import NotificationDB
```

Comment:
Should we perhaps remove this example completely in favor of `api-server.py`, or does this still provide a benefit?